com.woltlab.wcf/templates/headIncludeJavaScript.tpl

```diff
@@ -24,7 +24,8 @@
 	{/if}
 </script>
 
-<script src="{$__wcf->getPath()}js/WoltLabSuite/WebComponent.js"></script>
+<script src="{$phrasePreloader->getUrl($__wcf->language)}"></script>
+<script src="{$__wcf->getPath()}js/WoltLabSuite/WebComponent.js?v={@LAST_UPDATE_TIME}"></script>
```

Comment:
This is likely the wrong order: The WebComponent bundle needs to be loaded first for the called functions to exist.